manager2/src/app/home/home.component.ts

```diff
@@ -18,8 +18,10 @@ export class HomeComponent implements OnInit {
         this.config = {}
     }
 
-    ngOnInit() {
-        if (this.authService.isLoggedIn) {
+    async ngOnInit() {
+        let logged = await this.authService.isLoggedIn;
```

Comment:
isLoggedIn is a variable, not a fonction, so await should not do anything in this cased (logged will be set immediatly to true or false).

To change status when auth service log status change, you should subscribe to the login, and when callback arrive modify your log status.

You can see example in app.component.ts


    ngAfterViewInit() {
        this.loginSubscription = this.authService.$authStatus.subscribe((authenticated: boolean) => {
            setTimeout(() => {
                this.user = this.authService.userProfile;
                this.isLogged = authenticated;
            })
            if(authenticated) {